### DISCUSSION TITLE
Failing at the first hurdle - Pi  02W, "can't read /etc/rc.local no such file or directory"

### DISCUSSION THREAD

**Discussion by heyho74** [mcguirepr89/BirdNET-Pi#1269] [General]
Trying a fresh install on a brand new 02W.

Fresh install of RaspiOS-ARM64-Lite, updated, with the commands pasted in to expand the swap and switch off power saving, and I'm receiving "can't read /etc/rc.local no such file or directory"

`sudo sed -i 's/CONF_SWAPSIZE=100/CONF_SWAPSIZE=2048/g' /etc/dphys-swapfile &&
sudo sed -i 's/#CONF_MAXSWAP=2048/CONF_MAXSWAP=4096/g' /etc/dphys-swapfile &&
sudo sed -i '/^exit 0/i sudo iw wlan0 set power_save off' /etc/rc.local &&
sudo reboot
sed: can't read /etc/rc.local: No such file or directory
`

Hoping there is an easy fix so that I can move on to the actual install.

**Comment by Award1000** [comment]
I had the same problem. I found that I needed to create the file and add the text to it. I don't know if the standard installation for bookworm lite has changed but I didn't need to do this before. @heyho74. I used this page to help me do it - https://stackoverflow.com/questions/51434915/etc-rc-local-is-missing-from-my-headless-ubuntu-18-04
